### DISCUSSION TITLE
Hasty Getaway (GENERAL) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#27] [Card Balance] [answered]
Hasty Getaway (GENERAL) [C]
ANNIHILATED
Play during COMBAT. Target Combatant and all ATTACHMENT cards returns to owners hand. ATTACHMENTS owned by opponent are discarded.

It's like the original card game. It's madness in digital card game. 

Lot's of player will be crazy but we need to play this card to find a solution. 🤡

**Comment by TomekPL-1** [comment]
Yeah this card in all decks is super OP/ boring

**Reply by 21-ric** [reply]
Sadly I suggest to keep this card but to add a [DP] cost.

7 [DP] seems to be fair.

Banishment (BROTHERHOOD) [C]
ART
Cast as 2 Actions and 5 [DP]. One Warrior in play returns to its owner’s hand. Its Attachments are discarded.

Call For Backup! (GENERAL) [U]
Play on your defending combatant. For every 7 [DP] you spend, add the [F] and [S] of a different Warrior in your Squad to your combatant's.
➡️ add also [A] and [V].

**Comment by Sol-au-fein** [comment, marked as answer]
Hey, i have solution. Pay modified warrior's value.
Solved.

**Reply by 21-ric** [reply]
Base warrior's value is more interesting. 🤡

**Reply by Sol-au-fein** [reply]
Well, in that case it ignore Dark Symmetry powers or Strenght in Numbers. Not sure if It is all. Does Ki powers increasing V?

**Comment by TomekPL-1** [comment]
I would suggest shifting this card to one FACTION.

On turnament rules this card was Restricted to 1 copy per deck.

**Reply by Sol-au-fein** [reply]
It does not really fit to any corporation! Brotherhood/Black Legion on second thought.

**Reply by 21-ric** [reply]
It's stronger than "reassignment". 🤡

**Comment by 21-ric** [comment]
1) Restricted 👮‍♂️
2) Shift to one Faction 🤹‍♀️
3) Shift to one Faction an restricted 🤹‍♂️👮‍♂️
4) Play only to your warrior 💂‍♂️
5) Add a [DP] cost 💸
6) Find an other idea🕵️‍♀️
7) Don't touch this card 🤡

My preference goes to the number 5. @Sol-au-fein's idea is to pay modified warrior's value. I prefer to pay in [DP] the base value.

**Reply by Sol-au-fein** [reply]
Ok, base Value should be ok

**Comment by TomekPL-1** [comment]
I hope hasty getaway will not be generic. Reassignment in every deck is enought.

**Reply by 21-ric** [reply]
Or applies to both warriors. 🤐

**Reply by TomekPL-1** [reply]
That is different card 
![secondthoughts](https://user-images.githubusercontent.com/83038538/117574782-03a0d680-b0df-11eb-9711-3f86c7fb9fc0.jpg)

**Comment by nfortea** [comment]
It's to powerful. Shift to one faction too.
I think Play only to your warrior is enough.
In case would own by a faction without modifications, restricted one by deck.

**Reply by 21-ric** [reply]
Welcome aboard yz! 👋

**Comment by framerate** [comment]
Ok so... 

TODAY OR TOMORROW:
- Pay warriors base V in D
OR
- RESTRICTED (ew)

Is the top enough to satisfy?

LONG TERM -
- Move this card to a faction (or move other similar cards if this one stays General)

Thoughts?

**Reply by 21-ric** [reply]
If we keep "hasty getaway" like this :arrow_right: Restricted
But evryone will play 1 "hasty getaway" per deck :arrow_right: Faction 

Or

we add a DP cost (warriors base V)
"Banishment" cost 2 actions and 5DP.

**Reply by framerate** [reply]
... wait, so which do you prefer?

**Reply by 21-ric** [reply]
DP cost.

**Reply by Sol-au-fein** [reply]
Warrior's DP cost. This way you cant just make Alakhai or Doninic run away easily, while it will remain usable on low cost infantry.

**Reply by framerate** [reply]
Okay! I don't see anyone against this so I can patch it tomorrow and we'll see what people think!

**Reply by framerate** [reply]
This change is going live tonight 👍

**Comment by framerate** [comment]
Change started with BASE VALUE. Let me know feedback if you think MODIFIED is better

**Comment by Michalx11** [comment]
I think it's a good change... we could also change rarity from C to R

**Comment by Michalx11** [comment]
I think current version is still too OP. even with DP cost... my take is if you want to cancel combat then return your own warrior to hand not opponnents

**Reply by Sol-au-fein** [reply]
Im pretty happy with it now. Opponent must pay hefty cost and your warrior returns to your hand with all stuff on him. It is not OP anymore.

**Reply by Michalx11** [reply]
Totally disagree. If you play right DP cost wont be too much of a problem and if you are trying to make a comeback this kind of move is the nail to your coffin.... gataway is an escape from battle not a scare for pponnent to run and hide

**Reply by nfortea** [reply]
I'm with @Michalx11

**Reply by Sol-au-fein** [reply]
I admit, from cards name and picture, it would make sense to use it only on own warriors, but im ok either way.

**Reply by Michalx11** [reply]
Plus if you use it against opponnent's warriior (most players do) their attachment is discarded so it's really CANCEL WARRIOR instead of CAMCEL COMBAT

**Reply by Sol-au-fein** [reply]
False, all atachments are returned to owners hand

**Reply by 21-ric** [reply]
I have the feeling this card is balanced. We will see.

**Comment by Michalx11** [comment]
Ah yes I forgot it was changed. But that actually can be worsed as it clogged up your hand and you wont be able to draw as many (if any) cards next turn. And if opponnent used it on your warrior while you were attacking you are screwed... as you likely dont have enough DP and Actions left to quickly redeploy that warrior with all attachments... and if that was your last warrior you know you are going to be SABOTAGED mercilessly next turn and then scrambling to get back to game the following turn...  this nerf was a good attempt but it's just not enough...
I propose Pay 3DP to cancel combat and return OWN warrior and their attachments to hand
